examples/WebView/ReactNativeWebView.php

```diff
@@ -14,11 +14,11 @@ class ReactNativeWebView extends \LivingMarkup\Module
 {
     public function onRender()
     {
-        if($this->args['initiate'] == "true" && !empty($this->args['object'])) {
+        if ($this->args['initiate'] == "true" && !empty($this->args['object'])) {
             $object = $this->args['object']; // escape JSON
-            return "<script>
```

Comment:
I question returning a <body> tag as generally elements are replaced inside the a body.